pkg/pillar/cmd/volumemgr/dirs.go

```diff
@@ -8,24 +8,42 @@ import (
 )
 
 var (
-	downloaderObjTypes = []string{types.AppImgObj, types.BaseOsObj, types.CertObj}
+	downloaderObjTypes         = []string{types.AppImgObj, types.BaseOsObj, types.CertObj}
+	downloadWorkingDirTypes    = []string{"pending"}
+	downloadInProgressDirTypes = []string{"pending"}
+
+	verifierObjTypes         = []string{types.AppImgObj, types.BaseOsObj}
+	verifyWorkingDirTypes    = []string{"verifier", "verified"}
+	verifyInProgressDirTypes = []string{"verifier"}
 )
 
 func initializeDirs() {
 
+	// first the certs directory
+	if _, err := os.Stat(types.CertificateDirname); err != nil {
+		log.Debugf("initializeDirs: Create %s", types.CertificateDirname)
+		if err := os.MkdirAll(types.CertificateDirname, 0700); err != nil {
```

Comment:
This is also a directory that the verifier should own and have write access to (but we are waiting for a tweak to the design for the image signatures so it might change a bit). But it makes sense keeping this in verifier.

Reply:
Moved back to verifier.